neusomatic/include/Options.h

```diff
@@ -352,18 +414,24 @@ struct Options {
   std::string ref_;
   bool calculate_qual_stat_ = false;
   bool fully_contained_ = false;
-  float min_allele_freq_ = 0.01;
+  float snp_min_allele_freq_ = 0.01;
```

Comment:
Such constants should be declared somewhere else. Things like these would be good candidates for refactoring the code.